draft-packages/loading-spinner/docs/deprecated.LoadingSpinner.stories.tsx

```diff
@@ -8,13 +8,13 @@ import { figmaEmbed } from "../../../storybook/helpers"
 import { CATEGORIES } from "../../../storybook/constants"
 
 export default {
-  title: `${CATEGORIES.components}/Loading Spinner`,
+  title: `${CATEGORIES.components}/Loading Spinner (deprecated)`,
```

Comment:
For other cases where you do want to keep deprecated stories, there is a `CATEGORIES.deprecated` you can use here instead of `CATEGORIES.components` to group deprecated stories.

Reply:
We're actually keeping the files to keep the historical structure. We added some config so that files we rename to start with `deprecated.` are actually excluded from storybook and chromatic :)